aptos-move/framework/aptos-framework/sources/aptos_coin.move

```diff
@@ -41,8 +41,8 @@ module aptos_framework::aptos_coin {
 
         let (burn_cap, freeze_cap, mint_cap) = coin::initialize_with_parallelizable_supply<AptosCoin>(
             aptos_framework,
-            string::utf8(b"Aptos Coin"),
-            string::utf8(b"APT"),
+            string::utf8(b"Move Coin"),
```

Comment:
May want to rename the file and the Move package to `move_coin` as well, to forestall confusion.
If you do, be sure to find where the file is referenced and update the name there as well.